packages/graphile-build-pg/src/plugins/PgAllRows.js

```diff
@@ -58,7 +58,9 @@ export default (async function PgAllRows(
                 num => attributes.filter(attr => attr.num === num)[0]
               );
             const isView = t => t.classKind === "v";
-            const hasUniqueId = a => !!a.find(elem => elem.name === "uniqueId");
+            const uniqueIdAttribute = attributes.filter(
+              attr => attr.name === "uniqueId" || attr.name === "unique_id"
+            )[0];
```

Comment:
Prefer `.find(...)` over `.filter(...)[0]` as it does less work and consumes less memory.

Reply:
okay